src/wallet.js

```diff
@@ -1664,7 +1665,8 @@ Wallet.prototype.createAndSignTransaction = function(params, callback) {
       instant: params.instant,
       bitgoFee: bitgoFee,
       travelInfos: travelInfos,
-      estimatedSize: estimatedSize
+      estimatedSize: estimatedSize,
```

Comment:
we can use the shorthand syntax here and simplify `{ x: x, y: y }` to `{ x, y }`